packages/mobile/src/account/Settings.tsx

```diff
@@ -320,6 +320,25 @@ export class Account extends React.Component<Props, State> {
     this.setState({ showRevokeModal: false })
   }
 
+  // when change pin is clicked, we ensure that they have a pincode and then we navigate to pincode set screen with not verifying, and change pin is true
+
+  goToChangePin = () => {
+    ensurePincode()
+      .then((pinIsCorrect) => {
+        if (pinIsCorrect) {
+          this.props.navigation.navigate(Screens.PincodeSet, {
+            isVerifying: false,
+            changePin: true,
+          })
+        }
+      })
+      .catch((error) => {
+        Logger.error(`${'NavigationService'}@onPress`, 'PIN ensure error', error)
```

Comment:
nit: We don't need the `${}` if we're just writing a string. We could have something like:
`Logger.error(`NavigationService@onPress`, 'PIN ensure error', error)`

Reply:
sure!